controllers/cloud.redhat.com/providers/deployment/impl.go

```diff
@@ -72,6 +72,18 @@ func initDeployment(app *crd.ClowdApp, env *crd.ClowdEnvironment, d *apps.Deploy
 	}
 	d.Spec.ProgressDeadlineSeconds = common.Int32Ptr(600)
 
+	if !deployment.WebServices.Public.Enabled && deployment.WebServices.Private.Enabled {
+		if deployment.DeploymentStrategy != nil && deployment.DeploymentStrategy.PrivateStrategy != "" {
```

Comment:
Are we fine with just ignoring the strategy if it's public? Do we want some log or error to appear for debugging?

Reply:
I am personally fine with it since we have said in the docs it's ignored